btree/btree.c

```diff
@@ -1262,17 +1451,76 @@ static const struct seg_ops mem_seg_ops = {
 	.so_node_op_fini = &mem_node_op_fini
 };
 
+/**
+ *  Structure of the node in persistent store.
+ */
 struct ff_head {
-	struct m0_format_header ff_fmt;
-	struct node_header      ff_seg;
-	uint16_t                ff_used;
-	uint8_t                 ff_shift;
-	uint8_t                 ff_level;
-	uint16_t                ff_ksize;
-	uint16_t                ff_vsize;
-	struct m0_format_footer ff_foot;
+	struct m0_format_header ff_fmt;   /* Node Header */
+	struct node_header      ff_seg;   /* Node type */
+	uint16_t                ff_used;  /* Count of records */
+	uint8_t                 ff_shift; /* node size as pow-of-2 */
+	uint8_t                 ff_level; /* Level in Btree */
+	uint16_t                ff_ksize; /* Size of key in bytes */
+	uint16_t                ff_vsize; /* Size of value in bytes */
+	struct m0_format_footer ff_foot;  /* Node Footer */
+	/**
```

Comment:
Doxygen won't attach this anywhere. It would be better to put this information in comment right before ff_head.